integration-testing/config-local-host.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
maybe put these new files under the `/tests/` folder so it's less likely we slurp it into things like NPM/python/java packages